app/services/users.js

```diff
@@ -72,3 +72,16 @@ exports.listAll = (limit, offset) => {
     throw errors.defaultError(err.detail);
   });
 };
+
+exports.createOrUpdateAdmin = (user, newData) => {
+  return exports.getByEmail(user.email).then((foundUser) => {
```

Comment:
Same as I told you wit the update function, the `getByEmail` shouldn't be exported since no one outside the service is using it.